site/content/docs/concepts/pipelines.md

```diff
@@ -24,14 +24,14 @@ Want to learn more about CodePipeline? Check out their [getting started docs](ht
 Creating a Pipeline requires only three steps:
 
 1. Preparing the pipeline structure.
-2. Committing the generated `buildspec.yml`.
+2. Committing and pushing the generated `pipeline.yml` and `buildspec.yml`.
 3. Creating the actual CodePipeline.
 
 Follow the three steps below, from your workspace root:
 
 ```bash
 $ copilot pipeline init
-$ git add copilot/buildspec.yml && git commit -m "Adding Pipeline Buildspec" && git push
+$ git add copilot/pipeline.yml copilot/buildspec.yml && git commit -m "Adding Pipeline Buildspec" && git push
```

Comment:
```suggestion
$ git add copilot/pipeline.yml copilot/buildspec.yml copilot/.workspace && git commit -m "Adding Pipeline Buildspec" && git push
```

Reply:
Okay to change to just `git add copilot &&`...?

Reply:
Hidden files like .workspace don't get picked up automatically by `git add copilot/*` so it needs to be explicitly added, unfortunately. 

Could do `git add copilot/* copilot/.workspace && ...`

Reply:
or `git add --all copilot` also works!

Reply:
Ah, thanks. I'll change to your original suggestion bc it's the most clear for the reader.